web/gui/dashboard_info.js

```diff
@@ -1314,11 +1330,50 @@ netdataDashboard.context = {
 
     'ip.inerrors': {
         info: 'Errors encountered during the reception of IP packets. ' +
-            '<code>noroutes</code> (<code>InNoRoutes</code>) counts packets that were dropped because there was no route to send them. ' +
-            '<code>truncated</code> (<code>InTruncatedPkts</code>) counts packets which is being discarded because the datagram frame didn\'t carry enough data. ' +
-            '<code>checksum</code> (<code>InCsumErrors</code>) counts packets that were dropped because they had wrong checksum. '
+            '<b>noroutes</b> (<code>InNoRoutes</code>) counts packets that were dropped because there was no route to send them. ' +
+            '<b>truncated</b> (<code>InTruncatedPkts</code>) counts packets which is being discarded because the datagram frame didn\'t carry enough data. ' +
+            '<b>checksum</b> (<code>InCsumErrors</code>) counts packets that were dropped because they had wrong checksum. '
+    },
+    'ip.mcast': {
+        info: 'Total multicast traffic in the system.'
+    },
+    'ip.mcastpkts': {
+        info: 'Total transferred multicast packets in the system.'
+    },
+    'ip.bcast': {
+        info: 'Total broadcast traffic in the system.'
+    },
+    'ip.bcastpkts': {
+        info: 'Total transferred broadcast packets in the system.'
+    },
+    'ip.ecnpkts': {
+        info: 'Total number of received IP packets with ECN bits set in the system. '+
+        '<b>CEP</b> - congestion encountered. '+
+        '<b>NoECTP</b> - non ECN-capable transport. '+
```

Comment:
You wrote  `NoECTP` , but the description talks about `non ECN-Capable`, probably we need to rename the text inside bold for `NonECTP`.

Reply:
These are dimensions

<img width="476" alt="Screenshot 2021-09-22 at 17 38 28" src="https://user-images.githubusercontent.com/22274335/134364670-0b35fa9f-0512-47ef-be50-22a6174eda26.png">